Bikeini/screens/BikeInformation.js

```diff
@@ -322,6 +306,14 @@ class BikeInformation extends React.Component {
     // const { city, neighborhood } = location;    <- Ingen cykel har samma data-format .....
     const city = location;
     const neighborhood = location;
+    if (!dataLoaded) {
+      console.log('loading');
```

Comment:
Is this necessary for future troubleshooting or can it be removed? If possible I think it should be removed!